concepts/strings/about.md

```diff
@@ -0,0 +1,37 @@
+# About
+
+A `string` in Go is an immutable sequence of bytes, which don't necessarily have to represent characters.
```

Comment:
I am not certain if we want to specifically mention runes here. Perhaps not because you added another concept for `runes` that is supposed to be taught by `logs-logs-logs`.

Any preference @ekingery?

Reply:
Yes, as long as it is covered elsewhere, let's leave them out. In addition to the go101 link you recommend for the links.json below, I would defer to @ErikSchierboom as to whether this link is also appropriate: https://go.dev/blog/strings

It is much more comprehensive and does cover runes, which might be nice if people want to take a deeper dive with this concept.